cmd/version.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"fmt"
+	"runtime"
+	"strconv"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+// versionCmd represents the version command
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Get binary version.",
+	Long:  `Command to output version of a binary.`,
+	Run:   getVersion,
+}
+
+// Version string populated by releaser Job
+var (
+	Version string = "0.0.0"
+	// commitHash contains the current Git revision.
+	CommitHash string = "abcd"
+	// Timestamp contains the UnixTimestamp of the binary build.
+	Timestamp string = "1615361906"
```

Comment:
It should have 0, the value 1615361906 denotes ` Wednesday, 10 March 2021 07:38:26` which might confuse.